src/engine/ruleset/integrations/apache-http/test/error/error_expected.json

```diff
@@ -0,0 +1,227 @@
+      ],
+      "created": "2023-03-14T10:50:33.261Z",
+      "dataset": "apache-error",
+      "kind": "event",
+      "module": "apache-http",
+      "original": "[Tue Mar 14 10:50:33.261185 2023] [core:notice] [pid 3651:tid 139732022373440] AH00094: Command line: '/usr/sbin/apache2'",
+      "type": [
+        "info"
+      ]
+    },
+    "host": {
+      "id": "001"
+    },
+    "log": {
+      "level": "notice",
+      "logger": "core"
+    },
+    "message": "AH00094: Command line: '/usr/sbin/apache2'",
+    "process": {
+      "pid": 3651,
+      "thread": {
+        "id": 139732022373440
+      }
+    },
+    "service": {
+      "type": "apache"
+    },
+    "wazuh": {
+      "decoders": [
+        "apache-error"
+      ],
+      "location": "[001] (wazuh-agent-1) any->/test/apache2/error.log",
+      "origin": "/test/apache2/error.log",
+      "queue": 49,
+      "registered_ip": "any",
+      "source": "logcollector"
+    }
+  },
+  {
+    "message": "AH00094: Command line: '/usr/local/apache/httpd'",
+    "process": {
+      "pid": 11379
+    },
+    "ecs": {
+      "version": "8.8"
+    },
+    "event": {
+      "created": "2023-03-14T10:50:33.261Z",
+      "module": "apache-http",
+      "dataset": "apache-error",
+      "type": [
+        "info"
+      ],
+      "kind": "event",
+      "original": "[Tue Mar 14 10:50:33.261185 2023] [core:notice] [pid 11379] AH00094: Command line: '/usr/local/apache/httpd'",
+      "category": [
+        "web"
+      ]
+    },
+    "service": {
+      "type": "apache"
+    },
+    "wazuh": {
+      "location": "/var/log/apache2/access.log",
+      "queue": 49,
+      "source": "logcollector",
+      "decoders": [
+        "apache-error"
+      ]
+    },
+    "log": {
+      "level": "notice",
+      "logger": "core"
+    },
+    "TestSessionID": 1
+  },
+  {
+    "TestSessionID": 1,
+    "log": {
+      "logger": "rewrite",
+      "level": "trace1"
+    },
+    "wazuh": {
+      "queue": 49,
+      "source": "logcollector",
+      "location": "/var/log/apache2/access.log",
+      "decoders": [
+        "apache-error"
+      ]
+    },
+    "service": {
+      "type": "apache"
+    },
+    "message": "mod_rewrite.c(470): [client 127.0.0.1:8999] 192.168.0.1 - - [dev.test/sid#123456789][rid#ASD123485/test] applying pattern '^/import/?(.*)$' to uri '/'",
+    "process": {
+      "pid": 11379,
+      "thread": {
+        "id": 12312312313231
+      }
+    },
+    "ecs": {
+      "version": "8.8"
+    },
+    "event": {
+      "original": "[Tue Mar 14 10:50:33.261185 2023] [rewrite:trace1] [pid 11379:tid 12312312313231] mod_rewrite.c(470): [client 127.0.0.1:8999] 192.168.0.1 - - [dev.test/sid#123456789][rid#ASD123485/test] applying pattern '^/import/?(.*)$' to uri '/'",
+      "module": "apache-http",
+      "category": [
+        "web"
+      ],
+      "type": [
+        "info"
+      ],
+      "dataset": "apache-error",
+      "created": "2023-03-14T10:50:33.261Z",
+      "kind": "event"
+    },
+    {
```

Comment:
This json is poorly put together, it cannot be parsed. Please run the health tests